lib/rubocop/cop/rspec/let_setup.rb

```diff
@@ -28,7 +28,8 @@ module RSpec
       class LetSetup < Cop
         include RuboCop::RSpec::TopLevelDescribe
 
-        MSG = 'Do not use `let!` for test setup.'
+        MSG = 'Do not use `let!` for setup of objects' \
```

Comment:
WDYT of "Do not use `let!` to setup objects not referenced in tests."?

Reply:
Fine, and shorter.

Reply:
Changed